.github/workflows/test-conda.yml

```
@@ -14,23 +14,60 @@ on:
 
 jobs:
```

Ah, good catch, Matt. Here's a one-click version of his suggestion. 

```suggestion
defaults:
  run:
    shell: bash -l {0}

jobs:
```